### PR TITLE
add velocity for netcdf files

### PR DESCRIPTION
**Pull request by Krys1202**

### CODE REVIEW

File: pyschism/forcing/tides/tpxo.py [hunk 1]
```diff
@@ -66,17 +66,17 @@ def get_elevation(self, constituent, vertices):
 
     def get_velocity(self, constituent, vertices):
         raise NotImplementedError('Velocity not implemented for TPXO.')
```
Comment (jreniel) [inline pyschism/forcing/tides/tpxo.py:68]: We need to remove this line.

Comment (jreniel) [reply pyschism/forcing/tides/tpxo.py:68]: Also check line 102.

Comment (Krys1202) [reply pyschism/forcing/tides/tpxo.py]: I removed line 67 and updated line 102

**Comment by Krys1202** [comment]
Sorry maybe I shouldn't have closed the request right?

**Comment by Krys1202** [comment]
There was an error line 56, with the correction it's creating bctides.in for velocity

**Pull request merged**